using_ostree/examining.adoc

```diff
@@ -0,0 +1,65 @@
+[[using-ostree-examining]]
+= Examining Your System
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+
+=== Checking Your Current Ref
+
+OSTree "references" (abbreviated "refs") are textfiles that refer to particular
+commits. They are similar to git branches.
+
+The command `ostree refs` lists all refs available on the host. As an example
+of the output of this command:
+
+....
+$ ostree refs
+ostree/1/1/0
+ostree/1/1/1
+fedora-atomic:fedora/26/x86_64/updates/atomic-host
+....
+
+This shows the host has three refs, `ostree/1/1/0`, `ostree/1/1/1` and
+`fedora-atomic:fedora/26/x86_64/updates/atomic-host`. The refs starting with
+`ostree/` are local refs, the ref starting with `fedora-atomic:` is a remote
+ref.
+
+To check which ref is currently checked out, run `ostree admin status`:
+
+....
+$ ostree admin status
```

Comment:
should we also show the output of `rpm-ostree status` here and show the user that they essentially show the same output?

Reply:
as their isn't an `rpm-ostree` section, it makes sense to add that in here